display-scene-from-mobile-scene-package/README.metadata.json

```
@@ -0,0 +1,36 @@
+{
+    "category": "TODO",
+    "description": "Display a scene from a mobile scene package (.mspk)",
+    "formal_name": "DisplaySceneFromMobileScenePackage",
+    "ignore": false,
+    "images": [
+        "display-scene-from-mobile-scene-package.png"
+    ],
+    "keywords": [
+        "3d",
+        "mobile scene package",
+        "mspk",
+        "offline",
+        "scene",
+        "ArcGISScene",
+        "MobileScenePackage",
+        "SceneView"
+    ],
+    "language": "kotlin",
+    "provision_from": "TODO",
+    "provision_to": "TODO",
+    "redirect_from": "TODO",
+    "relevant_apis": [
+        "ArcGISScene",
+        "MobileScenePackage",
+        "SceneView"
+    ],
+    "snippets": [
+        "/src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/DownloadActivity.kt",
+        "/src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/MainActivity.kt",
+        "/src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/components/ComposeSceneView.kt",
+        "/src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/components/SceneViewModel.kt",
+        "/src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/screens/MainScreen.kt"
```

I realize the script probably generated this, but to build this in the sample viewer we need to remove the initial `/` path. 

```suggestion
        "src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/DownloadActivity.kt",
        "src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/MainActivity.kt",
        "src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/components/ComposeSceneView.kt",
        "src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/components/SceneViewModel.kt",
        "src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/screens/MainScreen.kt"
```

---

done